man/fi_domain.3.md

```diff
@@ -482,12 +482,19 @@ transfer operation.
   operations, or attempt to access outside of the target memory region
   will fail, resulting in a transmit error.
 
-When a resource management error occurs on an endpoint, the endpoint is
-transitioned into a disabled state.  Any operations which have not
-already completed will fail and be discarded.  For connectionless endpoints,
-the endpoint must be re-enabled before it will accept new data transfer
-operations.  For connected endpoints, the connection is torn down and
-must be re-established.
+When a resource management error occurs on an connected endpoint (e.g. unmatched
+RMA), the endpoint will transition into a disabled state and the connection torn
```

Comment:
The example is unnecessary since the description applies to all resource management errors.